JSDemos/Demos/Charts/TilingAlgorithms/description.md

```diff
@@ -1 +1,14 @@
-The TreeMap component supports three layout algorithms out of&nbsp;the box and provides the capability to&nbsp;implement your own algorithm. In&nbsp;this demo, you can change the applied algorithm at&nbsp;runtime using the drop-down menu above the component.
\ No newline at end of file
+Layout algorithms determine the position and size of tiles and groups. In this demo, you can use the drop-down menu below the component to apply different algorithms.
+
+The TreeMap component supports three layout algorithms out of the box:
+
+- **Squarified** (default)    
+This algorithm lays the items out so that the aspect ratio will be closer to 1. In other words, this algorithm tries to make items as square as possible. 
+
+- **Strip**     
+This algorithm is a modification of the *"Squarified"* algorithm. At the beginning, the algorithm has an available area divided into several strips and a set of items to distribute between the strips. Throughout the layout process, a current strip is maintained. For each item to be arranged, the algorithm checks whether or not adding the item to the current strip improves the average aspect ratios of the rectangles in the current strip. If so, the item is added to the current strip. Otherwise, it is added to the next strip.
```

Comment:
```suggestion
This algorithm is a modification of the *"Squarified"* algorithm. At the beginning, the algorithm has an available area divided into several strips and a set of items to distribute between the strips. Throughout the layout process, the current strip is maintained. For each item to be arranged, the algorithm checks whether or not adding the item to the current strip improves the average aspect ratios of the rectangles in the current strip. If so, the item is added to the current strip. Otherwise, it is added to the next strip.
```